### ISSUE TITLE
investigate creating raspberry pi images with earthly

### ISSUE THREAD

**Issue by alexcb** [earthly/earthly#1195]
It would be nice to investigate producing a raspberry pi image with a custom binary that could then be loaded onto an SD card via `dd`.

This could be done as a blog post.

**Comment by njskalski** [comment]
yeah, now I use a lot of time forking this https://github.com/raspi-alpine/builder and it takes a lot of time. Would love to switch to earthly, but I need a minimal example.

**Comment by alexcb** [comment]
I'll have to check that project out when I find some spare time.

I put together https://github.com/alexcb/raspberry-pi-pico-in-earthly for compiling code for the pico; however it's a much simpler process than a traditional pi image.
